src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -38,12 +39,11 @@ public class ScriptStartupTypeLocator : IWebJobsStartupTypeLocator
         private readonly IMetricsLogger _metricsLogger;
         private readonly Lazy<IEnumerable<Type>> _startupTypes;
         private readonly IOptionsMonitor<LanguageWorkerOptions> _languageWorkerOptions;
-
-        private static readonly ExtensionRequirementsInfo _extensionRequirements = DependencyHelper.GetExtensionRequirements();
+        private readonly ExtensionRequirementOptions _extensionRequirementOptions;
         private static string[] _builtinExtensionAssemblies = GetBuiltinExtensionAssemblies();
 
         public ScriptStartupTypeLocator(string rootScriptPath, ILogger<ScriptStartupTypeLocator> logger, IExtensionBundleManager extensionBundleManager,
-            IFunctionMetadataManager functionMetadataManager, IMetricsLogger metricsLogger, IOptionsMonitor<LanguageWorkerOptions> languageWorkerOptions)
+            IFunctionMetadataManager functionMetadataManager, IMetricsLogger metricsLogger, IOptionsMonitor<LanguageWorkerOptions> languageWorkerOptions, ExtensionRequirementOptions extensionRequirementOptions)
```

Comment:
Maybe I am missing something here, but why not do we not want to load in `IOptions< ExtensionRequirementOptions >` here?

Reply:
Have updated this to use `IOptions<ExtensionRequirementOptions>`